### DISCUSSION TITLE
Windows 11 File Explorer has a black outline box when choosing or last chosen file. Remove this ** ! (Image below)

### DISCUSSION THREAD

**Discussion by led9** [valinet/ExplorerPatcher#3491] [Q&A]
Hello,

After the newest Windows 11 update (3880, july_2024) I noticed there is a black outline that is always there when you just chosen a file or last choose the file.  Is an "accessibility feature" AND It doesn't turn off!

Image on page - https://answers.microsoft.com/en-us/windows/forum/all/image-below-windows-11-file-explorer-has-a-black/6674734c-956b-430a-9260-6b392eaec185

and on page https://answers.microsoft.com/en-us/windows/forum/all/after-the-latest-windows-11-update-i-see-a-black/81bd77cd-e6fc-4e54-a4ce-6590ef60a401

I want to be able to turn it off if I don't like it.

Search on the Internet showed that everyone is spitting, but there are no ways to turn it off.
All hope on ExplorerPatcher, valinet - for whom else? make a shutdown of this, please. (How you turn off rounded corners of the windows, for example).

**Comment by pyrates999** [comment]
The code in windows has to exist for it to be turned off.  Then maybe it can be added.

Keep an eye on windhawk as well for a mod that could disable this.

**Comment by GF-Huang** [comment]
Any news? I hate this shit black border.
